handlers/chat.js

```diff
@@ -191,42 +202,32 @@ handlers[Dota2.schema.EDOTAGCMsg.k_EMsgGCOtherJoinedChannel] = onOtherJoinedChan
 
 var onOtherLeftChannel = function onOtherLeftChannel(message) {
     /* Someone left a chat channel you're in. */
-    var otherLeft = Dota2.schema.CMsgDOTAOtherLeftChatChannel.decode(message);
-    var channel = this._getChannelById(otherLeft.channel_id);
+    var userWhoLeft = Dota2.schema.CMsgDOTAOtherLeftChatChannel.decode(message);
+    var channel = this._getChannelById(userWhoLeft.channel_id);
     // Check if it is me that left the channel
-    if ("" + otherLeft.steam_id === "" + this._client.steamID) {
-        if (this.debug) {
-            if (channel) {
+    if ("" + userWhoLeft.steam_id === "" + this._client.steamID) {
```

Comment:
While working on the fantasy stuff I found out that all these id's that are returned have a [Long](https://github.com/dcodeIO/long.js) type. This means that we can change this to the more readable:
```
if (userWhoLeft.steam_id.equals(this._client.steamID)) {
```